src/utils/daos.c

```diff
@@ -1116,6 +1123,8 @@ help_hdlr(struct cmd_args_s *ap)
 "	  list-objects     list all objects in container\n"
 "	  list-obj\n"
 "	  query            query a container\n"
+"	  get-prop         get a container's properties\n"
```

Comment:
Looks like I forgot to mention it in on-line help, so thank you !! ;-)

Reply:
No problem. :)